projects/sbb-esta/angular-business/chip/src/chip-input/chip-input.component.ts

```diff
@@ -0,0 +1,260 @@
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { SelectionModel } from '@angular/cdk/collections';
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostBinding,
+  Input,
+  OnInit,
+  Optional,
+  Output,
+  Self,
+  ViewChild
+} from '@angular/core';
+import { NgControl } from '@angular/forms';
+import {
+  AutocompleteComponent,
+  AutocompleteOriginDirective
+} from '@sbb-esta/angular-business/autocomplete';
+import { FormFieldControl } from '@sbb-esta/angular-core/forms';
+import { Subject } from 'rxjs';
+
+let nextUniqueId = 0;
+
+/** Change event object that is emitted when the chip-input value has changed. */
+export class SbbChipInputChange {
+  constructor(
+    /** Reference to the chip-input that emitted the change event. */
+    public source: ChipInputComponent,
+    /** Current value of the chip-input that emitted the event. */
+    public value: string[]
+  ) {}
+}
+
+@Component({
+  selector: 'sbb-chip-input',
+  templateUrl: './chip-input.component.html',
+  styleUrls: ['./chip-input.component.scss']
+})
+export class ChipInputComponent implements FormFieldControl<any>, OnInit {
+  /** Optional autocomplete Component */
+  @Input('sbbAutocomplete')
+  autocomplete: AutocompleteComponent;
+
+  /** Disables the chip input */
+  @Input()
+  @HostBinding('class.sbb-chip-input-disabled')
+  get disabled() {
+    return this._disabled;
+  }
+  set disabled(value: any) {
+    this._disabled = coerceBooleanProperty(value);
+  }
+  private _disabled = false;
+
+  /** Unique id of the element. */
+  @Input()
+  @HostBinding('attr.id')
+  get id(): string {
+    return this._id;
+  }
+  set id(value: string) {
+    this._id = value || this._uid;
+    this.stateChanges.next();
+  }
+  private _id: string;
+
+  /** Whether the component is required. */
+  @HostBinding('class.sbb-chip-input-required')
+  @Input()
+  get required(): boolean {
+    return this._required;
+  }
+  set required(value: boolean) {
+    this._required = coerceBooleanProperty(value);
+    this.stateChanges.next();
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-chip-input-active')
+  get _isActive() {
+    return !this.disabled && this.focused;
+  }
+
+  @ViewChild('chipInputTextfield', { static: false })
+  inputElement: ElementRef;
+
+  /** Value of the chip input control. */
+  get value(): any {
+    return this._value;
+  }
+  set value(newValue: any) {
+    if (newValue !== this._value) {
+      this._value = newValue;
```

Comment:
Assigning a new value should clear and update the selectionModel
Maybe simply call writeValue (writeValue should also clear the selectionModel)?